app/views/candidate_mailer/reinstated_offer.text.erb

```diff
@@ -2,7 +2,7 @@ Dear <%= @application_form.first_name %>
 
 <%= @provider_name %> has confirmed your deferred offer to study <%= @course_option.course.name_and_code %>.
 
-The course starts in <%= @course_option.course.start_date.to_fs(:month_and_year) %>.
+The course starts in <%= @course_option.course.start_date.advance(years: 1).to_fs(:month_and_year) %>.
```

Comment:
when a provider confirms a deferred offer we associate a `current_course_option` object with the application choice, for their new course. If we can access that instead, then we'll be able to use the start date.